core/domain/email_services.py

```diff
@@ -106,7 +106,9 @@ def _is_email_valid(email_address):
     if not stripped_address:
         return False
     # Regex for a valid email.
-    regex = r'^.+@(\[?)[a-zA-Z0-9-.]+.([a-zA-Z]{2,3}|[0-9]{1,3})(]?)$'
+    # Matches any characters before the at sign, a series of characters until a
```

Comment:
Not grammatically clear.

Matches any characters before the "@" sign, a series of characters until a ".", and then a series of characters after the period.